.babelrc

```diff
@@ -1,7 +1,11 @@
 {
-  "env": {
-    "test": {
-      "plugins": ["transform-es2015-modules-commonjs"]
-    }
-  }
+  "presets": [
+    "es2015",
```

Comment:
Maybe using [`babel-preset-env`](http://2ality.com/2017/02/babel-preset-env.html) with our targets might bring our bundle size down?

You may not need `transform-es2015-modules-commonjs` as I think that's the default in preset-env? though I would have expected webpack to handle the module side of things?